cypress/integration/creatorOnboardingFlows/creatorSettings.spec.js

```diff
@@ -234,14 +234,13 @@ describe('Admin -> Customization -> Config -> Images', () => {
       'not.exist',
     );
 
-    cy.findAllByRole('img', { name: /DEV\(local\)/i }).then((images) => {
```

Comment:
We've changed the way the logo renders in the Top left bar (in `_logo.html.erb`) , if there is no `resized_logo` then we can assume that we will have a community_name set instead. This test now reflects this.